core/server/worker/src/main/java/alluxio/worker/block/DefaultBlockWorker.java

```diff
@@ -349,6 +366,7 @@ public void requestSpace(long sessionId, long blockId, long additionalBytes)
   }
 
   @Override
+  @Deprecated
```

Comment:
why do you put this deprecated?

Reply:
Because it is marked `deprecated` in the interface definition. I copied here as a reminder to not adding test for this method.

Reply:
got it, thx!